### ISSUE TITLE
Display cover image on search with Tracking Service (MAL, etc.)

### ISSUE THREAD

**Issue by sahil-mohd** [inorichi/tachiyomi#1102]
Is it possible to add a link to the Manga's page if some tracking service is enabled?  
There are quite a few manga with similiar names that it is not easy to pick the right one (e.g., when I add "Orange" to my list, there are [two entries -- 1 and 3 --](https://myanimelist.net/manga.php?q=orange) with the same name: one is the manga I'm looking for, the other is a manhua). This can be avoided by displaying the cover image when searching. 

Another request is to place a link to the manga's page on the Tracking service card. This way it is easier to look for additional info on the manga instead of searching for it in an external browser. Let me know if this warrants another issue post.

**TL;DR**

- Display cover image when searching for manga in Tracking service
- Add link to manga's page on Tracking service on its card (in Tracking tab)

**Comment by inorichi** [comment]
This was already fixed.
